bin/export.py

```diff
@@ -40,18 +40,20 @@
     we = sp.array(h[2]['WE'][:])
     hep = sp.array(h[2]['HEALPID'][:])
 
+    head = h[1].read_header()
+    nt = head['NT']
+    np = head['NP']
+    rt_min = 0.
```

Comment:
what is rt_min used for? It doesn't seem to be written in the output...

Reply:
I just removed this variable